lib/testing/src/lib/core/actions/rest.api.service.ts

```diff
@@ -0,0 +1,107 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AlfrescoApiConfig, AlfrescoApi } from '@alfresco/js-api';
+import { browser } from 'protractor';
+import { Logger } from '../utils/logger';
+
+export class RestApiService {
+
+    apiService: AlfrescoApi;
```

Comment:
Is better to refactor the current ApiService to remove the compatibility layer

Reply:
Sure